sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample13_RecognizeInvoicesFromFile.cs

```diff
@@ -90,13 +100,20 @@ public async Task RecognizeInvoicesFromFile()
                                 }
                             }
 
-                            if (itemFields.TryGetValue("Quantity", out FormField itemQuantityField))
+                            if (itemFields.TryGetValue("Tax", out FormField itemTaxPriceField))
```

Comment:
as part of the work on https://github.com/Azure/azure-sdk-for-net/issues/15945 I am planning on updating the samples accordingly, so for now I just added the try-catch to the `tax` field